examples/plot_beginners_guide.py

```diff
@@ -0,0 +1,593 @@
+
+# setting and fitting the estimator on the source data
+base_classifier = SVC()
+base_classifier.fit(Xs, ys)
+
+# accuracy on target data
+accuracy = base_classifier.score(Xt, yt)
+
+# sphinx_gallery_start_ignore
+plt.figure(11, fig_size)
+source_target_comparison(
+    X, y, sample_domain, "Prediction on dataset", True, base_classifier
+)
+plt.tight_layout()
+print("Accuracy on target:", accuracy)
+# sphinx_gallery_end_ignore
+
+# %%
+# Now, we will illustrate what happens when using the Transfer Subspace Learning method
+adapted_clf = skada.TransferSubspaceLearning(base_classifier, n_components=1)
+adapted_clf.fit(X, y, sample_domain=sample_domain)
+accuracy = adapted_clf.score(Xt, yt)
+
+# sphinx_gallery_start_ignore
+plt.figure(12, fig_size)
+source_target_comparison(
+    X, y, sample_domain, "Prediction on dataset", True, adapted_clf
+)
+plt.tight_layout()
+print("\nAccuracy on target:", accuracy)
+# sphinx_gallery_end_ignore
+
+# %%
+# Results summary
+# ---------------
+#
+# Here is a summary of the different shifts and the domain adaptation methods used.
+
+# sphinx_gallery_start_ignore
+plt.figure(13, (17.5, 9))
+shift_list = ["covariate_shift", "target_shift", "conditional_shift", "subspace"]
+clfs = {
+    "covariate_shift": LogisticRegression().set_fit_request(sample_weight=True),
+    "target_shift": LogisticRegression().set_fit_request(sample_weight=True),
+    "conditional_shift": SVC(),
+    "subspace": SVC(),
+}
+shift_acc_before = {}
+shift_acc_after = {}
+
+for indx, shift in enumerate(shift_list):
+    X, y, sd = make_shifted_datasets(20, 20, shift, random_state=42)
+    Xs, Xt, ys, yt = skada.source_target_split(X, y, sample_domain=sd)
+    shift_name = shift.capitalize().replace("_", " ")
+
+    plt.subplot(4, 5, 2 + indx)
+    plot_full_data(X, y, shift_name)
+
+    plt.subplot(4, 5, 7 + indx)
+    plt.xlim(X[:, 0].min() - 1, X[:, 0].max() + 1)
+    plt.ylim(X[:, 1].min() - 1, X[:, 1].max() + 1)
+    plot_full_data(Xt, yt, None)
+
+    base_clf = clfs[shift]
+    base_clf.fit(Xs, ys)
+    shift_acc_before[shift_name] = base_clf.score(Xt, yt)
+    plt.subplot(4, 5, 12 + indx)
+    plot_full_data(Xt, yt, None)
+    decision_borders_plot(X, base_clf)
+
+    if shift == "covariate_shift" or shift == "target_shift":
+        adapted_clf = skada.DensityReweight(
+            base_estimator=base_clf, weight_estimator=KernelDensity(bandwidth=0.5)
+        )
+
+    elif shift == "conditional_shift":
+        adapted_clf = skada.OTMapping(base_clf)
+
+    elif shift == "subspace":
+        adapted_clf = skada.TransferSubspaceLearning(base_clf, n_components=1)
+
+    adapted_clf.fit(X, y, sample_domain=sd)
+    shift_acc_after[shift_name] = adapted_clf.score(Xt, yt)
+    plt.subplot(4, 5, 17 + indx)
+    plot_full_data(Xt, yt, None)
+    decision_borders_plot(X, adapted_clf)
+
+X, y, sd = make_shifted_datasets(20, 20, random_state=42)
+Xs, Xt, ys, yt = skada.source_target_split(X, y, sample_domain=sd)
+
+plt.subplot(4, 5, 1)
```

Comment:
For the summary plot I would remove the first columns. And maybe just have one line for source, one line for target, one line for train without adaptation and one last line for training with adaptation